packages/@aws-cdk/aws-autoscaling/lib/auto-scaling-group.ts

```diff
@@ -630,6 +731,29 @@ export class AutoScalingGroup extends AutoScalingGroupBase implements
     this.securityGroups.push(securityGroup);
   }
 
+  /**
+   * Emit all group metrics, these metrics describe the group rather than any of its instances
+   * see all available types https://docs.aws.amazon.com/autoscaling/ec2/userguide/as-instance-monitoring.html#as-group-metrics
+   * all group metrics are reported in granularity of 1Minute
+   * @default - disabled
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-as-metricscollection.html
+   */
+  public emitAllGroupMetrics() {
+    this.emitGroupMetrics();
+  }
+
+  /**
+   * Emit a specific subset of group metrics, these metrics describe the group rather than a single instance
+   * see all available types https://docs.aws.amazon.com/autoscaling/ec2/userguide/as-instance-monitoring.html#as-group-metrics
+   * all group metrics are reported in granularity of 1Minute
+   * to emit all group metrics use \`emitAllGroupMetrics\`
+   * @param metrics which groups metrics to collect
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-as-metricscollection.html
+   */
+  public emitGroupMetrics(...metrics: GroupMetric[]) {
```

Comment:
Shouldn't this be?

```suggestion
  public emitGroupMetrics(...metrics: GroupMetrics[]) {
``` 

How would one use this to emit different granularity for different metrics? Adding a `granularity` argument later on will be a breaking change.